script.js

```diff
@@ -1,98 +1,152 @@
+const body = document.querySelector("body");
 const gridContainer = document.querySelector(".grid-container");
 
-// use populateCells function to do what the name suggests 
-populateCells(16*16); 
-
+// use populateCells function to do what the name suggests
+populateCells(16 * 16);
 
 // Add event listener to the create button
-document.querySelector("#create").addEventListener("click",function(){
-    createGrid();
-});   
+document.querySelector("#create").addEventListener("click", function () {
+  createGrid();
+});
+
+// Add event listener to the reset button
+document.querySelector("#reset").addEventListener("click", function () {
+  const response = confirm("Are you sure?? Everything will be deleted");
+  if (response) {
+    body.style.setProperty("--rows", 16);
+    body.style.setProperty("--cols", 16);
+    populateCells(16 * 16);
+  }
+});
+
+// Add event listener to the remove grid button
+document.querySelector("#remove-grid").addEventListener("click", function () {
+  removeGrid();
+});
+
+// Add event listener to the clear button
+document.querySelector("#clear").addEventListener("click", function () {
+  const response = confirm("Are you sure?? Everything will be deleted");
+  if (response) clear();
+});
 
 let isSketching = false;
 let isErasing = false;
-// Add event listener (keydown) to the document to capture (s) and (e) keys in addtion to 
+// Add event listener (keydown) to the document to capture (s) and (e) keys in addtion to
 // (Enter) key
-document.addEventListener("keydown",function(event){
-    if(event.code ==="KeyS"){
-        isSketching = !isSketching;
-        isErasing = false;
-    }
-    if(event.code === "Enter"){
-        createGrid();
-        document.querySelector("#rows").blur(); 
-        document.querySelector("#cols").blur();
-    }
-    if(event.code === "KeyE"){
-       isErasing = !isErasing; 
-       isSketching = false;
-    }
-}); 
-
+document.addEventListener("keydown", function (event) {
+  if (event.code === "KeyS") {
+    isSketching = !isSketching;
+    isErasing = false;
+  }
+  if (event.code === "Enter") {
```

Comment:
I believe this will confuse the user, suppose I clicked enter by mistake while I am drawing. a pop up will be shown from nowhere. 
You can replace this by putting your inputs in a form and change the `create` button to input with type `submit`.

Reply:
Yes, I didn't see that coming. THANK YOU 👌👍
On it